cpp/test/CMakeLists.txt

```diff
@@ -308,24 +309,7 @@ if(BUILD_TESTS)
     NAME
     SPARSE_TEST
     PATH
-    sparse/add.cu
```

Comment:
Hey James, why are we removing all of these tests?

Reply:
Just recover it( it's for temporary debugging).